apps/web/screens/Organization/Syslumenn/Homestay.tsx

```diff
@@ -76,11 +81,67 @@ const Homestay: Screen<HomestayProps> = ({
     }),
   )
 
-  const [showCount, setShowCount] = useState(10)
+  const [showCount, setShowCount] = useState(PAGE_SIZE)
   const [query, _setQuery] = useState(' ')
 
   const setQuery = (query: string) => _setQuery(query.toLowerCase())
 
+  const csvStringProvider = () => {
+    return new Promise<string>((resolve, reject) => {
+      if (homestays) {
+        const COLUMN_SEPARATOR = ','
+        const ROW_SEPARATOR = '\n'
+        const HEADER_COLUMNS = [
```

Comment:
Perhaps we'd like to define these constants at the top of the file instead of inside a function that's inside a react component. Just a thought, one can argue that these constants are only utilised here so therefore they should live next to the place they're used.